src/components/map/Map.tsx

```diff
@@ -11,6 +11,7 @@ import { MapMarker } from '../../types';
 type Props = {
   geometryTourData?: LatLng[];
   isMaximizeButtonVisible?: boolean;
+  isPOI?: boolean;
```

Comment:
i think this should be renamed to something that makes it clear to be applied for a map overview with multiple markers, right? a POI is one point of interest but the condition we implement is for "a map with a lot markers" or "a map with just one marker". `isPOI` does not fit that in my eyes.

Reply:
How about `isMultipleMarkersMap`? I think it's right for this place.

Reply:
i like it

Reply:
I changed it.

https://github.com/smart-village-solutions/smart-village-app-app/commit/34ff12eaf22df69c02cde6db188b3c78f214a07c